meadow-client/src/MainFrame.purs

```diff
@@ -625,9 +626,9 @@ mainHeader = div_ [ div [classes [btnGroup, pullRight]] (makeLink <$> links)
                   , h1 [class_ $ ClassName "main-title"] [text "Meadow"]
                   ]
   where
-  links = [ Tuple "Getting Started" "https://testnet.iohkdev.io/plutus/get-started/writing-contracts-in-plutus/"
-          , Tuple "Tutorial" "https://github.com/input-output-hk/plutus/blob/master/plutus-tutorial/tutorial/Tutorial/02-wallet-api.md"
-          , Tuple "API" "https://input-output-hk.github.io/plutus/"
+  links = [ -- Tuple "Getting Started" "https://testnet.iohkdev.io/plutus/get-started/writing-contracts-in-plutus/"
+          -- , Tuple "Tutorial" $ "https://github.com/input-output-hk/plutus/blob/" <> gitHead <> "/plutus-tutorial/tutorial/Tutorial/02-wallet-api.md"
```

Comment:
Why commented out?

Reply:
These links are incorrect, I wanted them left to signify that they should be fixed. I will open an issue for this.